content/en/blog/_posts/2020-05-21-wsl2-dockerdesktop-k8s.md

```diff
@@ -153,7 +153,7 @@ And as sources are always important to mention, we will follow (partially) the h
 
 ```bash
 # Download the latest version of KinD
-curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/v0.7.0/kind-$(uname)-amd64
+curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/v0.7.0/kind-linux-amd64

```

Comment:
this was correct, but at some point case insensitive download stopped working, and in this context only linux is relevant so.